docs/tutorials/advanced/aqua/optimization/1_optimization_problems.ipynb

```diff
@@ -25,6 +25,798 @@
     "    - build a model in docplex with integer variables and constraints, cast to QUBO. \n",
     "    - introduce convenience converter \"to_qubo\"..."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Converters for `QuadraticProgram`\n",
+    "To solve an optimization prbolem with quantum computer, a corresponding Ising Hamiltonian for the optimization problem is needed to be created. To do so, firstly, it's necessary to convert it into with a specific form, more precisely Unconstrained Binary Optimization form. For that purpose, Qiskit has three converters. They are `InequalityToEqualityConverter`, `IntegerToBinaryConverter`  and `PenalizeLinearEqualityConstraints`.With these converters, optimization problems can be easily and automatically converted into one with a specific form to solve quantum computers. The following is a list of three converters. The details of each converter will be discussed later with examples.\n",
```

Comment:
here is a typo

Reply:
Ah I see, my actual PR is `2_converters_for_quadratic_programs.ipynb` file, not `1_optimization_problems.ipynb` file. That's why I couldn't find it. It's fixed in the latest version in Stefan's repo as well. https://github.com/stefan-woerner/qiskit/blob/qiskit-optimization/docs/tutorials/advanced/optimization/2_converters_for_quadratic_programs.ipynb  Anyway, thank you for the comment.